src/components/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import axios from 'axios';
+import UserProfile from './UserProfile';
+import {
+    loadUserPending,
+    loadUserSuccess,
+    loadUserFailed,
+    updateUserPosts,
+} from '../../redux/userSlice';
+import { useCookies } from 'react-cookie';
```

Comment:
its a good practice to import you packages in a file in the below order
 - first import the package(s) which  are related to react package
 - then import the 3rd party or npm packages
 - then import your local packages or the components which have been created in application

Reply:
fixed